eth_tester/backends/pyevm/main.py

```diff
@@ -131,12 +131,15 @@ def get_default_account_keys(quantity=None):
 
 
 @to_tuple
-def get_account_keys_from_mnemonic(mnemonic, quantity=None):
+def get_account_keys_from_mnemonic(mnemonic, quantity=None, hd_path_override=None):
     keys = KeyAPI()
     seed = seed_from_mnemonic(mnemonic, "")
     quantity = quantity or 10
     for i in range(0, quantity):
         hd_path = HDPath(f"m/44'/60'/0'/{i}")
+        if hd_path_override is not None:
```

Comment:
nit: I'm seeing we use `hd_path` as the kwarg in all other methods. We could use that here as well (instead of `hd_path_override`) and, if it's `None`, set the internal value, also so we don't set the `hd_path` variable twice with each iteration.